src/main/java/langley/tasks/Deadline.java

```diff
@@ -0,0 +1,32 @@
+package langley.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
```

Comment:
Maybe writing descriptive header comments for all public classes/methods could improve the readability of the code?
The following suggestions are extracted from [here](https://se-education.org/guides/conventions/java/intermediate.html)
You MUST write header comments for all classes, public methods. But they can be omitted for the following cases:
- Getters/setters
- When overriding methods (provided the parent method's Javadoc applies exactly as is to the overridden method)